code/mathematical-algorithms/Check_is_square/FindingISquare.cpp

```diff
@@ -1,25 +1,51 @@
-#include <iostream>
+#include <bits/stdc++.h>
 using namespace std;
-int main()
+
+typedef long long int lld;
```

Comment:
Why the _d_ at the end. Why not _i_?

Reply:
In C, we have 'lld' as format specifier for long long int . I'm just sticking to that convention of naming.